### DISCUSSION TITLE
Re-clustering per cell type

### DISCUSSION THREAD

**Discussion by paulasoler** [stuart-lab/signac#565] [Q&A] [answered]
Once we defined the main clusters in our dataset, we decided to split the Seurat/Signac object by each of them and now I would like to re-clustering the data. The main question is: Do I have to repeat the normalization step for each cell type?. I share my proposed code to do it:

seurat <- seurat %>%
  RunTFIDF(method = 2) %>% 
  FindTopFeatures(min.cutoff = "q0") %>%
  RunSVD()
  RunHarmony(group.by.vars = "gem_id", reduction = "lsi") %>%
  RunUMAP(reduction = "harmony", dims = 1:40) %>%
  FindNeighbors(reduction = "harmony", dims = 1:40)

* seurat -> seurat object per each cell type.

Is it fine?

**Comment by timoast** [comment, marked as answer]
When you subset the cells the inverse document frequency value (IDF) used in TF-IDF will change, so I would recommend re-running `RunTFIDF()` and `RunSVD()` on the subset data.

**Comment by paulasoler** [comment]
Thanks a lot for your answer. However, when I did the re-clustering using the previously shared code, I saw a main cluster that determine the cell of interest and around it a kind of satellite groups. Have you ever seen this before ? It may be due to the normalization method that I used, number of PCs, doublets ...?. 

![image](https://user-images.githubusercontent.com/3385292/116682909-2b8d8d00-a9af-11eb-9d32-1b2d1d9a93ac.png)
